test/end-to-end/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "end-to-end",
+  "version": "0.0.0",
+  "description": "weldr-web end-to-end automation test",
+  "scripts": {
+    "test": "mocha -R xunit -O output=result.xml"
+  },
+  "dependencies": {
```

Comment:
Not in dependencies, but in devDependencies.
I have to develop and run cases in top level environment, so I need these package in devDependencies in the top level package.json file.